src/extensionConfiguration.ts

```diff
@@ -0,0 +1,16 @@
+import { loadConfigurationForExtension } from '@codingame/monaco-editor-wrapper/dist/features/extensionConfigurations'
+import staticOptions, { LanguageClientId } from './staticOptions'
+
+export async function loadExtensionConfigurations (forLanguageClientIds: LanguageClientId[], nonMutualizableOnly: boolean): Promise<void> {
+  const extensionConfigurationToLoad = new Set<string>()
+  for (const languageClientId of forLanguageClientIds) {
+    const config = staticOptions[languageClientId]
+    if (!config.mutualizable || !nonMutualizableOnly) {
+      (config.vscodeExtensionIds ?? []).forEach(extensionId => {
```

Comment:
just wondering: is it better to use `(config.vscodeExtensionIds ?? []).forEach` instead of the shortest `config.vscodeExtensionIds?.forEach` ?

Reply:
Both are probably fine I guess?

Reply:
b852cd19dc80b2e90dec45bab7f8e27033a78c32